egs/swbd_fisher/run.sh

```diff
@@ -15,67 +15,52 @@ fisher_dirs="/export/corpora3/LDC/LDC2004T19/fe_03_p1_tran/ /export/corpora3/LDC
 
 local/fisher_data_prep.sh $fisher_dirs
 
-fold_dev_opt=
```

Comment:
@wantee, we need to have a way to pass this --fold-dev-into option in via the top-level train_lm.py- can you please keep this in the script and show how you'd pass it in?